docker/main.Dockerfile

```diff
@@ -25,49 +25,49 @@ RUN apt install -y libapr1-dev libgmp-dev libpcre3-dev libboost-regex-dev
 # Rust.
 # https://forge.rust-lang.org/infra/other-installation-methods.html#standalone-installers
 ARG RUST_VERSION=1.81.0
-RUN wget --no-verbose https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz
-RUN wget --no-verbose https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz.asc
-RUN gpg --verify rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz.asc rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz
-RUN tar -xzf rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz
-RUN ./rust-${RUST_VERSION}-x86_64-unknown-linux-gnu/install.sh
-RUN rm rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz.asc
+RUN wget --no-verbose https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz \
+ && wget --no-verbose https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz.asc \
+ && gpg --verify rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz.asc rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz \
+ && tar -xzf rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz \
+ && ./rust-${RUST_VERSION}-x86_64-unknown-linux-gnu/install.sh \
+ && rm -rf rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz.asc rust-${RUST_VERSION}-x86_64-unknown-linux-gnu
```

Comment:
Nit, can you have 4 spaces between start of line and `&&`?

Reply:
I fixed it in a new commit